components/cms/KiboProductCarousel.vue

```diff
@@ -1,70 +1,56 @@
-        </template>
-        <SfCarouselItem
-          v-for="(product, i) in products"
-          :key="`carousel-item-${productGetters.getProductId(product)}`"
-          class="carousel__item"
-        >
-          <KiboProductCard
-            :key="productGetters.getProductId(product)"
-            :style="{ '--index': i }"
-            :score-rating="3"
-            :max-rating="5"
-            wishlist-icon=""
-            is-in-wishlist-icon=""
-            :is-in-wishlist="false"
-            :title="productGetters.getName(product)"
-            :image="productGetters.getCoverImage(product)"
-            :show-add-to-cart-button="false"
-            :regular-price="`$${productGetters.getPrice(product).regular}`"
-            :special-price="
-              productGetters.getPrice(product).special && productGetters.getPrice(product).special
-            "
-            :link="localePath(getProductLink(productGetters.getProductId(product)))"
-            image-width="12.563rem"
-            image-height="12.563rem"
-            class="products__product-card"
-          />
-        </SfCarouselItem>
-      </SfCarousel>
+  <div class="carousel-wrapper" v-if="!loading && products.length > 0">
+    <div class="header">
+      <p class="title">{{ title }}</p>
+      <SfLink>{{ $t("Shop All") }}</SfLink>
+    </div>
+    <div class="product-carousel-container">
+      <KiboProductCard
+        v-for="(product, i) in products"
+        :key="productGetters.getProductId(product)"
+        :style="{ '--index': i }"
+        :score-rating="3"
+        :max-rating="5"
+        wishlist-icon=""
+        is-in-wishlist-icon=""
+        :is-in-wishlist="false"
+        :title="productGetters.getName(product)"
+        :image="productGetters.getCoverImage(product)"
+        :show-add-to-cart-button="false"
+        :regular-price="`$${productGetters.getPrice(product).regular}`"
+        :special-price="
+          productGetters.getPrice(product).special && productGetters.getPrice(product).special
+        "
+        :link="localePath(getProductLink(productGetters.getProductId(product)))"
+        image-width="12.563rem"
+        image-height="12.563rem"
+        class="products__product-card"
+      />
     </div>
   </div>
 </template>
 <script>
-import { SfCarousel, SfArrow } from "@storefront-ui/vue"
+import { SfLink } from "@storefront-ui/vue"
 import { useProductSearch, useUiHelpers } from "@/composables"
 import { productGetters } from "@/lib/getters"
 export default {
   components: {
-    SfCarousel,
-    SfArrow,
+    SfLink,
   },
   props: {
     productCodes: {
-      type: Array
+      type: Array,
+      default: () => [],
     },
     title: {
       type: String,
       default: "",
     },
   },
   setup(props) {
-    const { search, result } = useProductSearch(`top-sellers`)
+    const { search, result, loading } = useProductSearch(`top-sellers`)
```

Comment:
add a key or id prop to use in place of 'top-sellers'
otherwise having more than one instance of this component on a page could display the wrong results

Reply:
Resolved